commit de8af64dad24b35599617dc887e5bccc5397985c (refs/changes/77/4877/4)
Author: Robert Bogart <Robert.bogart@joyent.com>
Date:   2018-10-17T21:29:31+00:00 (12 months ago)
    
    MANTA-3854 Mako sometimes fails to create manifests due to limitations in 'find'
    Reviewed by: Robert Mustacchi <rm@joyent.com>
    Reviewed by: Trent Mick <trent.mick@joyent.com>
    Approved by: Trent Mick <trent.mick@joyent.com>

diff --git a/targets.json.in b/targets.json.in
index a39048f..b6a0f78 100644
--- a/targets.json.in
+++ b/targets.json.in
@@ -1223,10 +1223,12 @@ cat <<EOF
     "build_platform": "20151126T062538Z",
     "image_name": "manta-storage",
     "image_description": "Manta storage",
-    "// image_uuid": "triton-origin-multiarch-15.4.1@1.0.1",
-    "image_uuid": "04a48d7d-6bb5-4e83-8c3b-e60a99e0f48f",
+    "// image_uuid": "triton-origin-multiarch-18.1.0@1.0.1",
+    "image_uuid": "b6ea7cb4-6b90-48c0-99e7-1d34c2895248",
     "pkgsrc": [
-      "pcre-8.38"
+      "pcre-8.42",
+      "findutils-4.6.0nb1",
+      "gawk-4.1.4nb1"
     ],
     "repos": [
       {"url": "git@github.com:joyent/manta-mako.git"}
